### PR TITLE
Politer language for call notifications for private calls 

### PR DESCRIPTION
**Pull request by jakobroehrl**
Fix https://github.com/nextcloud/spreed/issues/4354

### CODE REVIEW

File: lib/Notification/Notifier.php [hunk 1]
```diff
@@ -563,7 +563,7 @@ protected function parseCall(INotification $notification, Room $room, IL10N $l):
 			if ($user instanceof IUser) {
 				if ($this->notificationManager->isPreparingPushNotification() || $room->hasSessionsInCall()) {
```
Comment (nickvergessen) [inline lib/Notification/Notifier.php:564]: this will create a conflict with #4324 so lets delay this until the other one is in, to avoid tones of rebases

**Review by nickvergessen — APPROVED** [review]

File: lib/Notification/Notifier.php (hunk 1 above)
Comment (PVince81) [reply lib/Notification/Notifier.php:564]: the PR in question was merged, no conflict has appeared :-D

Comment (nickvergessen) [reply lib/Notification/Notifier.php]: Yeah, it is 2 lines apart and that seems to be enough...

**Comment by ChristophWurst** [comment]
:heart: :handshake: :smile:

**Pull request merged**